ColorPatterns/Scenes/GameOver/GameOverViewModel.swift

```diff
@@ -27,6 +27,7 @@ struct GameOverViewModel {
             if error != nil {
                 print(error?.localizedDescription as Any)
             } else {
+                //TODO: Remove prints when not needed
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/162adab2e9334135ce6ea45f840937e07368c165/ColorPatterns/Scenes/GameOver/GameOverViewModel.swift#L30"></span>TODOs should be resolved (Remove prints when not needed).<br />
<code>todo</code> <code>GameOverViewModel.swift:30</code></td>
    </tr>
  </tbody>
</table>